packages/flutter/test/services/text_formatter_test.dart

```diff
@@ -79,7 +80,7 @@ void main() {
       );
       expect(
         FilteringTextInputFormatter(RegExp('o+'), allow: false, replacementString: '*').formatEditUpdate(testOldValue, selectedIntoTheWoods),
-        const TextEditingValue(text: 'Int* the W**ds', selection: TextSelection(baseOffset: 11, extentOffset: 14)),
+        const TextEditingValue(text: 'Int* the W*ds', selection: TextSelection(baseOffset: 11, extentOffset: 13)),
```

Comment:
Is this behavior change part of the fix?

Reply:
Yeah the double "o" was separated by the caret so the filtering was applied separately. Hopefully nobody is relying on this behavior.